samples/quickstart.js

```diff
@@ -20,15 +20,15 @@
 const PubSub = require('@google-cloud/pubsub');
 
 // Your Google Cloud Platform project ID
-const projectId = 'YOUR_PROJECT_ID';
```

Comment:
So AFAIK all projects use 'YOUR_PROJECT_ID'. I'd prefer to keep it consistent over projects.